src/lib/Components/Inbox/ActiveBids/ActiveBid.tsx

```diff
@@ -186,7 +184,7 @@ interface RelayProps {
       href: string | null
       is_live_open: boolean | null
     } | null
-    active_bid: {
+    most_recent_bid: {
```

Comment:
Yayy to TypeScript -- made this change immediately jump out all across call-sites.